qa/tasks/cephfs/test_sessionmap.py

```diff
@@ -230,3 +230,49 @@ def test_session_evict_blocklisted(self):
 
         self.mount_a.kill_cleanup()
         self.mount_a.mount_wait()
+
+    def test_session_evict_non_blocklisted(self):
+        """
+        Check that mds evicts without blocklisting client
+        """
+
+        self.fs.set_ceph_conf('mds', 'mds session blocklist on evict', False)
+        self.fs.mds_restart()
+        status = self.fs.wait_for_daemons()
+
+        self.fs.set_max_mds(2)
+        status = self.fs.wait_for_daemons()
+
+        self.fs.set_ceph_conf('client', 'client reconnect stale', True)
```

Comment:
`mds_defer_session_stale` defaults to `true`, i hope this doesn't hamper the test case

Reply:
you referred to `client_reconnect_stale`, i was pointing to `mds_defer_session_stale`, both are a bit different

Reply:
Okay, my mistake. No I won't.